src/Parsers/DSL/Dates/ManualDateParserTrait.php

```diff
@@ -0,0 +1,27 @@
+<?php
+
+namespace Rapidez\StatamicQueryBuilder\Parsers\DSL\Dates;
```

Comment:
Could you put the traits in a seperate namespace? `Rapidez\StatamicQueryBuilder\Parsers\DSL\Dates\Traits`